agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4748,6 +4741,27 @@ bool slave_thread::handle_ap_capability_query(Socket *sd, ieee1905_1::CmduMessag
     return true;
 }
 
+bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received CLIENT_STEERING_REQUEST_MESSAGE , mid=" << std::dec << int(mid);
+    // build ACK message CMDU
+    auto cmdu_tx_header = cmdu_tx.create(mid, ieee1905_1::eMessageType::ACK_MESSAGE);
+
+    if (!cmdu_tx_header) {
+        LOG(ERROR) << "cmdu creation of type ACK_MESSAGE, has failed";
+        return false;
+    }
+    auto error_code_tlv = cmdu_tx.addClass<wfa_map::tlvErrorCode>();
```

Comment:
AFAIU, we need to add error_code_tlv only if the station received in the command is not connected to the AP.
I don't think we want to simulate error scenarios.
What actually does missing is sending back CMDU of BTM REPORT (0x9c).